Week1/compairingInfinities.js

```diff
@@ -0,0 +1,13 @@
+// store diffrent types of data in one array, and compare the infinitives
+let allTypesInsideArray = ['abdullah ', 33 , true]
+console.log(allTypesInsideArray);
+
+
+console.log('2 examples for compairing infinities')
+if(6/0 === 10/0){
+    console.log(true);
+} else{
+    console.log(false);
+}
+
+console.log(6/0 == 10/0);
```

Comment:
here you use `==` and in the other if statement you use `===` is this done on purpose:)?

And it's better to store the results of `6/0` and `10/0` in variables and then compare them, it makes your code a bit more readable!

Reply:
I wanted to see both option (== / ===)
I updated with two variables. 
thanks